public/service-worker.js

```diff
@@ -0,0 +1,153 @@
+
+// Set a name for the current cache
+var cacheName = 'v1'; 
+
+// Default files to always cache
+var cacheFiles = [
+  '/',
+  '/__/firebase/7.7.0/firebase-app.js',
+  '/__/firebase/7.7.0/firebase-auth.js',
+  '/__/firebase/7.7.0/firebase-firestore.js',
+  '/__/firebase/init.js',
+  '/index.html',
+  '/js/',
+  '/js/components/charts.js',
+  '/js/components/dropdown.js',
+  '/js/components/picker.js',
+  '/js/components/pulldown.js',
+  '/js/components/toggleswitch.js',
+  '/js/components/wheel.js',
+  '/js/shared/animate.js',
+  '/js/shared/auth.js',
+  '/js/shared/date.js',
+  '/js/shared/globals.js',
+  '/js/shared/handle_tick.js',
+  '/js/shared/helpers.js',
+  '/js/shared/route.js',
+  '/js/shared/store.js',
+  '/js/shared/user.js',
+  '/js/shared/uuid.js',
+  '/js/templates/partials/footer.js',
+  '/js/templates/partials/section_grade-selector.js',
+  '/js/templates/partials/section_otc.js',
+  '/js/templates/partials/section_progress.js',
+  '/js/templates/partials/status_ticker.js',
+  '/js/templates/page_groups.js',
+  '/js/templates/page_home.js',
+  '/js/templates/page_history.js',
+  '/js/templates/page_login.js',
+  '/js/templates/page_settings.js',
+  '/js/templates/page_signup.js',
+  '/js/templates/page_statistics.js',
+  '/js/app.js',
+  /* CSS */
+  '/css/animations.css',
+  '/css/buttons.css',
+  '/css/fonts.css',
+  '/css/footer.css',
+  '/css/groups.css',
+  '/css/history.css',
+  '/css/horizontal_menu.css',
+  '/css/legends.css',
+  '/css/login.css',
+  '/css/main.css',
+  '/css/otc.css',
+  '/css/status_ticker.css',
+  '/css/variables.css'
+
+]
+
+
+self.addEventListener('install', function(e) {
+    console.log('[ServiceWorker] Installed');
```

Comment:
This file has different indentation than the others. Copy pasted from somewhere? Should we intent it to match the rest of the code base?